### DISCUSSION TITLE
database migration question 9 to 9

### DISCUSSION THREAD

**Discussion by eliteproxy7** [directus/directus#9729] [Archive]
is there a way to set up a migration script that automatically syncs up two databases that are both running same version of Directus on the same version of DB

**Comment by rijkvanzanten** [comment]
Syncs the whole database, or only the schema? If the latter, you could rely on a flow where you're exporting the schema using `directus schema snapshot` on the main instance, after which you'll import that schema on bootstrapping of your instances with `directus schema apply`
